test/beacon/Voucher.test.ts

```diff
@@ -48,26 +77,51 @@ describe('Voucher', function () {
             requester,
             anyone,
         ] = await ethers.getSigners();
-        const beacon = await voucherUtils.deployBeaconAndImplementation(admin.address);
+        // Deploy PoCo mock and VoucherHub.
         iexecPocoInstance = await new IexecPocoMock__factory()
             .connect(admin)
             .deploy()
             .then((x) => x.waitForDeployment());
         iexecPoco = await iexecPocoInstance.getAddress();
-        const voucherHub = await voucherHubUtils.deployHub(
+        beacon = await voucherUtils.deployBeaconAndImplementation(admin.address);
```

Comment:
Why move this line here and not like before on top of line 81 => will remove some diff for the PR stat (?)

Reply:
The goal is to separate different parts:
* First, deploy PoCo
* Then, deploy Voucher contracts.
* Fund VoucherHub
* ...etc
Also, this PR stats are quite good so I don't think it's needed here.